SeedService/src/SeedService_service.cc

```diff
@@ -317,6 +320,12 @@ namespace mu2e {
     state_.clear();
   }
 
+  void SeedService::postEvent             (art::Event const& ev, art::ScheduleContext context) {
+    if(verbosity_ > 1) {
+      std::cout << "SeedService postEvent Event " << ev.id() << " current seed " << currentSeed_ << std::endl;
```

Comment:
Is it useful to keep this?  This just prints out the next available (or last used?) seed from the SeedService.  It has nothing to do with the state of any engine.

Reply:
my bad, I didn't intend to include this mod in this PR, it was an experiment that failed.  I am backing it out now.